crates/storage-query-datafusion/src/deployment/schema.rs

```diff
@@ -15,8 +15,8 @@ use crate::table_macro::*;
 use datafusion::arrow::datatypes::DataType;
 
 define_table!(deployment(
-    id: DataType::LargeUtf8,
-    ty: DataType::LargeUtf8,
-    endpoint: DataType::LargeUtf8,
-    created_at: DataType::Date64,
+    id "The ID of the service deployment.": DataType::LargeUtf8,
+    ty "The type of the endpoint. Either `http` or `lambda`.": DataType::LargeUtf8,
+    endpoint "The address of the endpoint. Either HTTP URL or Lambda ARN.": DataType::LargeUtf8,
+    created_at "Timestamp indicating the deployment registration time.": DataType::Date64,
```

Comment:
The syntax adds a lot of inline noise. Maybe an alternative is to move this to be a comment on every field instead.

Reply:
I like this idea. It is a bit similar to how clap does it for the command line options, right?

Reply:
I will try to finish this PR while slinkydeveloper is on vacation to avoid PR rot.